TombEngine/Game/effects/tomb4fx.cpp

```diff
@@ -1578,7 +1578,7 @@ void UpdateShockwaves()
 
 				if (LaraItem->HitPoints > 0)
 				{
-					if (sw->flags & 3)
+					if (sw->damage & 3)
```

Comment:
This condition doesn't make sense now. You should use `if (sw->damage)` instead.

Reply:
fixed